### DISCUSSION TITLE
Planowanie sprintu 2

### DISCUSSION THREAD

**Discussion by FilipParyz** [FilipParyz/UE_Kat#9] [General]
Hej, zapraszam do dyskusji na temat planowania Sprintu 2.
Proszę o sugestie zadań, które powinniśmy wykonać w tym sprincie. Każdy z nas powinien podać co najmniej jedną/dwie sugestie zadań do wykonania (polecam unikać ogólników i rozpisać parę mniejszych i łatwiejszych do wykonania zadań).
Po rozpisaniu zadań postaramy się je zaplanować używając tablicy Gnatt w Projekcie. Poniżej spiszę swoje zadania, można się zainspirować.

**Comment by FilipParyz** [comment]
Zadania do wykonania:
- [x] Wykończenie funkcjonalnego menu
- [x] Dodanie ekranu "Credits"
- [x] Dodanie ekranu wyboru gier
- [x] Dodanie ekranu do przeglądania osiągnięć i wyników
- [ ] Zmiana wyglądu i dodanie grafik
- [x] Dodanie gier do menu wyboru gier
- [ ] Integracja stworzonych gier z menu głównym.

**Comment by DorotaSzota** [comment]
Gra Pong:
- [ ] Dodać ograniczenia/ramki wokół ekranu tak, żeby paletka nie wychodziła poza ekran
- [ ] Dodać wyświetlania punktów
- [ ] Dodać balans progresywnego zwiększania prędkości piłeczki

**Reply by DorotaSzota** [reply]
Zrobiłam rework Ponga (nazwa Table Tennis). Teraz są dwie paletki i można grać w dwie osoby.

**Comment by owcen** [comment]
Gra Warcaby:
- [ ] naprawić sterowanie
- [ ] poprawić wygląd planszy
- [ ] dopracować poruszanie pionkiem

**Comment by mmisiewiczz** [comment]
Gra Saper:

- [x] stworzyć planszę gry
- [x] stworzyć funkcję wybierania pól
- [x] stworzyć funkcję losowego rozmieszczania bomb
- [x] dodać zaznaczanie bomb po trafieniu na czerwono
- [x] dodać opcję wyświetlania ilości trafionych pól
- [x] dodać opcje rozpoczęcia gry ponownie i wyjścia z gry
- [x] dodać grę Tiktaktoe i Saper do repozytorium

**Comment by ghost** [comment]
Gra Wąż:

- [ ]  Dodać grafiki do okienek pop-up

- [ ]  Spróbować zmienić ikonę w rogu okienek na wężyka

- [ ]  Usprawnić działanie i przekierowywanie po wybraniu opcji w okienkach 

- [ ] zamykanie gry po pewnym czasie po przegranej
